.github/workflows/build-phar.yml

```diff
@@ -61,3 +61,8 @@ jobs:
         with:
           name: pie-${{ github.sha }}.phar
           path: pie.phar
+      - name: Store PHAR as artifact
```

Comment:
This is not necessary, the step before already stores the PHAR

Reply:
NM, was just updated

Reply:
Yeah, my bad, I did not change restoring artifact during rebase and it failed, I thought storing was missing. Fixed already.